packages/ui/src/components/SelectInputV2/SelectBar.tsx

```diff
@@ -84,36 +77,71 @@ const StyledInputWrapper = styled(Stack)<{
   }
   &[data-state='neutral'] {
     border: 1px solid ${({ theme }) => theme.colors.neutral.border};
+
+    &:not([data-disabled="true"]):not([data-readonly="true"]):active {
```

Comment:
It seems like the disabled is still not ok, the text is still dark in color 🤔 
<img width="824" alt="Screenshot 2024-09-17 at 11 36 14" src="https://github.com/user-attachments/assets/d8f139e7-4b23-44b1-8d9c-dbd2555e359b">

Reply:
it is corrected here: https://github.com/scaleway/ultraviolet/pull/4238